content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -289,6 +289,106 @@ The Pod which is already running correctly would be killed by `activeDeadlineSec
 The name of each app and init container in a Pod must be unique; a
 validation error is thrown for any container sharing a name with another.
 
+### Sidecar containers and restartPolicy
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Thanks to a newly available feature gate named `SidecarContainers`, each
```

Comment:
Please avoid terms like 'newly available' because things decay very soon in this community.

Reply:
Google have some nice [tips](https://developers.google.com/style/timeless-documentation) on that topic.